File: src/Plotly.NET/SpaceFrame.fs
namespace Plotly.NET

open DynamicObj
open System

type Spaceframe () =
    inherit DynamicObj () 

    static member init 
        (
            ?Fill: float,
            ?Show: bool
        ) =
            Spaceframe()
            |> Spaceframe.style
                (
                    ?Fill       = Fill,
                    ?Show       = Show
                )

    static member style
        (
            ?Fill: float,
            ?Show: bool
        ) =

            fun (spaceframe: Spaceframe) ->
                
                Fill        |> DynObj.setValueOpt spaceframe "fill"
                Show        |> DynObj.setValueOpt spaceframe "show"

File: src/Plotly.NET.CSharp/ObjectAbstractions/TraceObjects/SpaceFrame.cs

using Microsoft.FSharp.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Plotly.NET.CSharp;

public static partial class TraceObjects
{
    public static Plotly.NET.TraceObjects.Spaceframe Spaceframe
        (

            Optional<double> Fill = default,
            Optional<bool> Show = default
        ) => 
            Plotly.NET.TraceObjects.Spaceframe.init(

                Fill: Fill.ToOption(),
                Show: Show.ToOption()
            );
}

